src/main/java/ru/skypro/homework/controller/AdsController.java

```diff
@@ -0,0 +1,108 @@
+package ru.skypro.homework.controller;
+
+import lombok.RequiredArgsConstructor;
+import lombok.extern.slf4j.Slf4j;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.MediaType;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.*;
+import org.springframework.web.multipart.MultipartFile;
+import ru.skypro.homework.dto.AdCreateOrUpdate;
+import ru.skypro.homework.dto.AdDto;
+import ru.skypro.homework.dto.AdInfo;
+import ru.skypro.homework.dto.AdsAll;
+
+import java.io.IOException;
+
+@Slf4j
+@CrossOrigin(value = "http://localhost:3000")
+@RestController
+@RequiredArgsConstructor
+@RequestMapping("/ads")
+public class AdsController {
+
+    /**
+     * Добавление объявления
+     *
+     * @param image
+     * @param ad
+     * @return
+     */
+    @PostMapping(consumes = MediaType.MULTIPART_FORM_DATA_VALUE)
+    public ResponseEntity<AdDto> addAd(@RequestPart("image") MultipartFile image, @RequestPart AdDto ad) throws IOException {
```

Comment:
```suggestion
    public ResponseEntity<AdDto> addAd(@RequestPart("image") MultipartFile image, @RequestPart("properties") AdDto ad) throws IOException {
```
название второй части properties должно быть, иначе не будет работать (см. спецификацию)